src/vaadin-chart-series.html

```diff
@@ -169,6 +169,7 @@
 
             /**
              * The name of the series as shown in the legend, tooltip etc.
+             * @type {string}
              */
             title: {
```

Comment:
`title` has no default value so should include `| undefined`.

Reply:
This is intended. See `Findings` in the PR description.